ts/components/ui/Markdown/index.tsx

```diff
@@ -363,7 +380,8 @@ class Markdown extends React.PureComponent<Props, State> {
     animated: boolean = false,
     onError?: (error: any) => void,
     cssStyle?: string,
-    useCustomSortedList: boolean = false
+    useCustomSortedList: boolean = false,
+    avoidTextSelect: boolean = false
```

Comment:
```suggestion
    avoidTextSelection: boolean = false
```
Could it be a better name?

Reply:
Yes sounds good, I previously told it but then i truncated the name